docs/reference/configuration.md

```diff
@@ -107,6 +107,8 @@ This reference uses `.` to denote the nesting of values.
 + `email.username` `(string)` - A string representing the username of the specific account at the SMTP. It's recommended to provide this value at runtime from a secret vault of some sort.
 + `email.password` `(string)` - A string representing the password of the account. It's recommended to provide this value at runtime from a secret vault of some sort.
 + `email.sender` `(string)` - A string representing the email account which is set as the sender.
++ `email.rate_limit` `(float)` - A float representing the amout emails that can be sent per second.
```

Comment:
This is a really hard number to select correctly.  Indeed, to set the default you explicitly inverted a useful number.  I would suggest:
1. `email.limit_every` (Duration): Average time to wait between sending emails.
1. `email.burst` (int): Maximal burst of emails before applying `limit_every`.

Then you can compute the rate limit param as something like `float(time.Second) / float(config.Email.LimitEvery)`.

WDYT?

Reply:
accepted and done